order-service/src/main/java/com/microcommerce/orderservice/data/entity/Order.java

```diff
@@ -33,6 +34,9 @@ public class Order {
     @Column(name = "order_number")
     private String orderNumber;
 
+    @Column(name = "user_id")
```

Comment:
First to merge Bojana's PR for drawing the base model and then to rebase changes from develop in order to expand the base model with the user id field.

Reply:
Done